src/Chainweb/Miner/Pact.hs

```diff
@@ -89,17 +89,20 @@ data Miner = Miner !MinerId !MinerKeys
     deriving stock (Eq, Ord, Show, Generic)
     deriving anyclass (NFData)
 
+-- NOTE: These JSON instances are used (among other things) to embed Miner data
+-- into the Genesis Payloads. If these change, the payloads become unreadable!
+--
```

Comment:
It may be policy if this instance can be changed or not. Internally then old field names are more compact/efficient, but externally the UX is bad. Both `chainweb-node` and `chainweb-miner` utilize these instances for configuration.